internal/operator/cluster/clusterlogic.go

```diff
@@ -798,3 +798,28 @@ func ScaleClusterDeployments(clientset kubernetes.Interface, cluster crv1.Pgclus
 	}
 	return
 }
+
+// waitFotDeploymentReady waits for a deployment to be ready, or times out
+func waitForDeploymentReady(clientset kubernetes.Interface, namespace, deploymentName string, periodSecs, timeoutSecs time.Duration) error {
+	ctx := context.TODO()
+
+	// set up the timer and timeout
+	// first, ensure that there is an available Pod
+	timeout := time.After(timeoutSecs * time.Second)
+	tick := time.NewTicker(periodSecs * time.Second)
```

Comment:
I see that this is moved from elsewhere, but multiplying a `time.Duration` by `time.Second` is going to cause trouble. The arguments `periodSecs` and `timeoutSecs` should be `int` or used directly here.

My preference is the latter. I suggest changing the following in `internal/operator/cluster/pgadmin.go`:

```go
const (
	deployTimeout = 60 * time.Second
	pollInterval  = 3 * time.Second
)
```

And similar in `internal/operator/cluster/rolling.go`.

The trouble:
```go
	fmt.Println(3 * time.Second)
	fmt.Println(3 * time.Second * time.Second)
```
```
3s
833333h20m0s
```